verification/curator-service/ui/src/components/landing-page/LandingPage.tsx

```diff
@@ -114,11 +113,23 @@ const MoreInformationLinks = ({
     version,
     env,
 }: StyleProps & { version: string; env: string }) => {
+    const releaseNotesUrl =
```

Comment:
this can be extracted into a utility function along with the duplicate in `App/index.tsx`